taskana-adapter-test/src/main/resources/processes/simple_user_task_process_with_complex_variables.bpmn

```diff
@@ -3,7 +3,7 @@
   <bpmn:process id="simple_user_task_with_complex_variables_process" isExecutable="true">
```

Comment:
In this process, it would be fine to have another service task after the first user task that verifies that the variables came back from the user task as expected....

Reply:
Added the additional service task service task